_opensearch/rest-api/index-apis/clear-index-cache.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Clear Index or Data Stream Cache
+parent: Index APIs
+grand_parent: REST API reference
+nav_order: 320
+---
+
+## Clear index or data stream cache
+
+Clears the caches of one or more indices. For data streams, the API clears the caches of the stream’s backing indices.
```

Comment:
change all instances of "indices" to "indexes" (decision was made to use the "indexes" term throughout, unless it's a coded name)